### PR TITLE
added java files

### PR DESCRIPTION
**Pull request by conorg763**

### CODE REVIEW

File: Java/JavaCodeSamples/src/main/java/com/truestock/Main.java [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+        }
+
+        return response.body();
+
+    }
+
+    private static void getStoreOptions() {
+        HttpClient client = HttpClient.newHttpClient();
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(URI.create("https://janus-api-preprod.truestock.io/user/store-location/options"))
```
Comment (John-Doherty01) [inline Java/JavaCodeSamples/src/main/java/com/truestock/Main.java]: URL make global string `https://janus-api-preprod.truestock.io` then within the function `URL + 'user/store-location/options'`

File: Java/JavaCodeSamples/src/main/java/com/truestock/Main.java [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+                .header("Content-Type", "application/json")
+                .header("Authorization", "Bearer " + token)
+                .GET()
+                .build();
+
+        sendRequest(client, request);
+
```
Comment (John-Doherty01) [inline Java/JavaCodeSamples/src/main/java/com/truestock/Main.java]: return something from this function

File: Java/JavaCodeSamples/src/main/java/com/truestock/Main.java [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+    private static void getAllStoreLocation(int pageNumber) {
+        HttpClient client = HttpClient.newHttpClient();
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(URI.create(url + "/user/store-location/all?page=" + pageNumber))
+                .header("Content-Type", "application/json")
+                .header("Authorization", "Bearer " + token)
+                .GET()
+                .build();
+
+        sendRequest(client, request);
```
Comment (John-Doherty01) [inline Java/JavaCodeSamples/src/main/java/com/truestock/Main.java]: return something for this function

File: Java/JavaCodeSamples/src/main/java/com/truestock/Main.java [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+                .uri(URI.create(url + "/forecast/all?page=" + pageNumber))
+                .header("Content-Type", "application/json")
+                .header("Authorization", "Bearer " + token)
+                .GET()
+                .build();
+
+        client.sendAsync(request, HttpResponse.BodyHandlers.ofString())
+                .thenApply(HttpResponse::body)
+                .thenAccept(System.out::println)
+                .join();
```
Comment (John-Doherty01) [inline Java/JavaCodeSamples/src/main/java/com/truestock/Main.java]: return something for this function

File: Java/JavaCodeSamples/src/main/java/com/truestock/Main.java [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+    private static void getForecastInput(int forecastId) {
+        HttpClient client = HttpClient.newHttpClient();
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(URI.create(url + "forecast/get/input/" + forecastId + "/"))
+                .header("Content-Type", "application/json")
+                .header("Authorization", "Bearer " + token)
+                .GET()
+                .build();
+
+        sendRequest(client, request);
```
Comment (John-Doherty01) [inline Java/JavaCodeSamples/src/main/java/com/truestock/Main.java]: return something from this function

File: Java/JavaCodeSamples/src/main/java/com/truestock/Main.java [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+    private static void getForecastResultJSON(int forecastId) {
+        HttpClient client = HttpClient.newHttpClient();
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(URI.create(url + "forecast/get/result/" + forecastId + "/"))
+                .header("Content-Type", "application/json")
+                .header("Authorization", "Bearer " + token)
+                .GET()
+                .build();
+
+        sendRequest(client, request);
```
Comment (John-Doherty01) [inline Java/JavaCodeSamples/src/main/java/com/truestock/Main.java]: return something for this function

File: Java/JavaCodeSamples/src/main/java/com/truestock/Main.java (hunk 1 above)
Comment (conorg763) [reply Java/JavaCodeSamples/src/main/java/com/truestock/Main.java]: Added.

File: Java/JavaCodeSamples/src/main/java/com/truestock/Main.java (hunk 2 above)
Comment (conorg763) [reply Java/JavaCodeSamples/src/main/java/com/truestock/Main.java]: added fix.

File: Java/JavaCodeSamples/src/main/java/com/truestock/Main.java (hunk 3 above)
Comment (conorg763) [reply Java/JavaCodeSamples/src/main/java/com/truestock/Main.java]: returned

File: Java/JavaCodeSamples/src/main/java/com/truestock/Main.java (hunk 4 above)
Comment (conorg763) [reply Java/JavaCodeSamples/src/main/java/com/truestock/Main.java]: resolved.

**Pull request merged**